modules/apps/portal-cache/portal-cache-impl/src/test/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImplTest.java

```diff
@@ -168,6 +168,8 @@ private void _testPutAndGetNullModel(boolean serialized) {
 	private static final MockedStatic<FrameworkUtil>
 		_frameworkUtilMockedStatic = Mockito.mockStatic(FrameworkUtil.class);
 
+	private final BundleContext _bundleContext =
+		SystemBundleUtil.getBundleContext();
 	private ClassLoader _classLoader;
```

Comment:
Hey @GabrielHenriquedev, this commit seems to be in the wrong order. You didn't change any code related to EntityCache in the previous ones.